lifelines/plotting.py

```diff
@@ -332,78 +332,102 @@ def plot_estimate(
       ax: a pyplot axis object
     """
 
-    if censor_styles is None:
-        censor_styles = {}
-
-    if (loc is not None) and (iloc is not None):
-        raise ValueError("Cannot set both loc and iloc in call to .plot().")
-
-    set_kwargs_ax(kwargs)
-    set_kwargs_color(kwargs)
-    set_kwargs_drawstyle(kwargs)
-
-    if estimate is None:
-        estimate = cls._estimate_name
-
-    if estimate == "hazard_":
-        if bandwidth is None:
-            raise ValueError("Must specify a bandwidth parameter in the call to plot_hazard.")
-        estimate_ = cls.smoothed_hazard_(bandwidth)
-        confidence_interval_ = cls.smoothed_hazard_confidence_intervals_(bandwidth, hazard_=estimate_.values[:, 0])
-    else:
-        estimate_ = getattr(cls, estimate)
-        confidence_interval_ = getattr(cls, "confidence_interval_")
-
+    config = PlotEstimateConfig(
+        cls,
```

Comment:
I would avoid using this as a variable name especially in the context of a class method. 
https://stackoverflow.com/a/7554899/6304433

Reply:
Good point, Ill fix this and create a new PR